change-kube-version-offline.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+if [ $1 == "" ]
+then
+    echo "Usage: ./change-kube-ver.sh <version>";
+    exit
+fi
+version=$1
+dir="roles/kubespray-defaults/defaults/main.yaml"
```

Comment:
We just need to write `kube_version` on inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml instead of having a script to update the default value.
I don't think it is worth maintaining the separated script for just doing that.

Reply:
Agree 
@Keyz078 Thank you for this but the need for this is too low to burden ourselves with another script